extensions/skyportal/static/js/components/ZTFAlertsByCoords.jsx

```diff
@@ -0,0 +1,372 @@
+
+  const theme = useTheme();
+  const darkTheme = theme.palette.type === "dark";
+
+  const alerts = useSelector((state) => state.alertsByCoords);
+
+  const makeRow = (alert) => {
+    return {
+      objectId: alert?.objectId,
+      candid: alert?.candid,
+      jd: alert?.candidate.jd,
+      ra: alert?.candidate.ra,
+      dec: alert?.candidate.dec,
+      drb: alert?.candidate.drb,
+      fid: alert?.candidate.fid,
+      mag: alert?.candidate.magpsf,
+      emag: alert?.candidate.sigmapsf,
+      programid: alert?.candidate.programid,
+    };
+  };
+
+  let rows = [];
+
+  if (alerts !== null && !isString(alerts) && Array.isArray(alerts)) {
+    rows = alerts.map((a) => makeRow(a));
+  }
+
+  useEffect(() => {
+    const fetchAlerts = async () => {
+      const data = await dispatch(Actions.fetchAlertsByCoords({ ra, dec, radius }));
+      setIsFetched(true);
+    };
+
+    if (!isFetched) {
+      fetchAlerts();
+    }
+  }, [dispatch, isFetched, ra, dec, radius]);
+
+  const classes = useStyles();
+
+  // This is just passed to MUI datatables options -- not meant to be instantiated directly.
+  const renderPullOutRow = (rowData, rowMeta) => {
+    const colSpan = rowData.length + 1;
+    const alertData = alerts[rowMeta.dataIndex];
+    const thumbnails = [
+      {
+        type: "new",
+        id: 0,
+        public_url: `/api/alerts/ztf/${alertData.objectId}/cutout?candid=${alertData.candid}&cutout=science&file_format=png`,
+      },
+      {
+        type: "ref",
+        id: 1,
+        public_url: `/api/alerts/ztf/${alertData.objectId}/cutout?candid=${alertData.candid}&cutout=template&file_format=png`,
+      },
+      {
+        type: "sub",
+        id: 2,
+        public_url: `/api/alerts/ztf/${alertData.objectId}/cutout?candid=${alertData.candid}&cutout=difference&file_format=png`,
+      },
+    ];
+
+    return (
+      <TableRow data-testid={`alertRow_${alertData.candid}`}>
+        <TableCell
+          style={{ paddingBottom: 0, paddingTop: 0 }}
+          colSpan={colSpan}
+        >
+          <Grid
+            container
+            direction="row"
+            spacing={3}
+            justify="center"
+            alignItems="center"
+          >
+            <Grid item>
+            <ThumbnailList
+              thumbnails={thumbnails}
+              ra={alertData.candidate.ra}
+              dec={alertData.candidate.dec}
+              displayTypes={["new", "ref", "sub"]}
+            />
+            </Grid>
+          </Grid>
+        </TableCell>
+      </TableRow>
+    );
+  };
+
+  const options = {
+    selectableRows: "none",
+    expandableRows: true,
+    expandableRowsOnClick: true,
+    renderExpandableRow: renderPullOutRow,
+    elevation: 1,
+    sortOrder: {
+      name: "jd",
+      direction: "desc",
+    },
+  };
+
+  const columns = [
+    {
+      name: "objectId",
+      label: "Object ID",
+      options: {
+        filter: true,
+        sort: true,
+        sortDescFirst: true,
+      },
```

Comment:
One more request @acrellin: could you make the objectId's clickable (or turn into obvious buttons?) and point them at `/alerts/ztf/<objectId>`?

Reply:
Done: 
![alerts_by_coords_table_button_objid](https://user-images.githubusercontent.com/7230285/112554834-46ebf380-8d84-11eb-9b92-dc08008592d4.gif)